config/dev.exs

```diff
@@ -1,7 +1,7 @@
 use Mix.Config
 
 database_url =
-  System.get_env("DATABASE_URL") || "ecto://postgres:password:password@db:5432/postgres"
+  System.get_env("DATABASE_URL") || "postgresql://postgres:postgres@localhost:5432/workbench_dev"
```

Comment:
I think this will cause problems with docker compose. I'll take a deeper look at how this is structured and get back to you over the weekend.